octodns/provider/ns1.py

```diff
@@ -192,6 +192,9 @@ class Ns1Provider(BaseProvider):
     }, {
         'config': {},
         'filter': u'geofence_regional'
+    }, {
+        'config': {},
+        'filter': u'geofence_country'
```

Comment:
I believe this will need to detect the old set and "update" it to the new style in case anyone has already created records with the old filter chains. If this just changes I believe those existing records will cause exceptions when people try to sync anything they created before.